runtime/doc/treesitter.txt

```diff
@@ -378,6 +378,15 @@ get_string_parser({str}, {lang}, {opts})                 *get_string_parser()*
                     {lang}  The language of this string
                     {opts}  Options to pass to the created language tree
 
+is_ancestor({dest}, {source})                                  *is_ancestor()*
+                Determines whether {dest} is a ancestor of {source}.
+
+                Parameters: ~
+                    {dest}    The destination node (the potential ancestor)
+                    {source}  The source node (the potential child)
```

Comment:
```suggestion
                    {source}  The source node (the potential descendant)
```

Reply:
(will be overwritten by the doc generation script anyway)

Reply:
Oh Ok I just found out about `gen_vimdoc.py` ^^ In that case I guess I'll make a final commit with its output for `runtime/doc/treesitter.txt`